src/lib/learn-client/api/collection/formatting.ts

```diff
@@ -44,6 +45,9 @@ export function formatCollection(collection: ApiCollection): Collection {
 		level,
 		category: category || undefined,
 		tutorials: tutorials.map(formatToTutorialLite),
+		...(next_collection
+			? { nextCollection: formatToCollectionLite(next_collection) }
+			: {}),
```

Comment:
We should probably default to `undefined`?

Reply:
Why should we default to undefined? This only adds a `next_collection` property if it exists

Reply:
For sure, its up to you since its the same end result but the suggestion was more for consistency with the other learn client formatting functions.